File: Android/Garima0812S_GarimaShukla_2125ec1114_2/GettingStarted/inverseofaNumber.java
import java.util.*;

public class Main{

public static void main(String[] args) {
  // write your code here
  Scanner sc=new Scanner(System.in);
  int n=sc.nextInt();
  int i=0;
  int p=1;
  while(n>0)
  {
      int dig=n%10;
      n=n/10;
      i+=p*Math.pow(10,dig-1);
      p++;
  }
  System.out.println(i);
 }
}

File: Android/tiwari-aditi_AditiTiwari_2125ec1138_2/GettingStarted/InverseOfANumber.cpp
#include<iostream>
#include<math.h>
using namespace std;

int main(){
    int start=1,n,dig,i =0,mult;
    cin>>n;
    
    while(n!=0){
        dig = n%10;
        dig--;
        mult = pow(10,dig);
        i+= start*mult;
        n= n/10;
        start++;
    }
    
    cout<<i;
    return 0;
 }